extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlIntegrationTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.cdc.mysql;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.hazelcast.jet.accumulator.LongAccumulator;
+import com.hazelcast.jet.cdc.AbstractIntegrationTest;
+import com.hazelcast.jet.cdc.ChangeRecord;
+import com.hazelcast.jet.cdc.Operation;
+import com.hazelcast.jet.cdc.RecordPart;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.StreamSource;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Objects;
+import org.junit.Rule;
+import org.testcontainers.containers.MySQLContainer;
+
+import static com.hazelcast.jet.Util.entry;
+import static org.testcontainers.containers.MySQLContainer.MYSQL_PORT;
+
+public class AbstractMySqlIntegrationTest extends AbstractIntegrationTest {
+
+    protected static final String DATABASE = "testDb";
+    protected static final String SINK_MAP_NAME = "resultsMap";
+
+    @Rule
+    public MySQLContainer<?> mysql = new MySQLContainer<>("debezium/example-mysql:1.2")
+            .withUsername("mysqluser")
+            .withPassword("mysqlpw");
+
+    protected MySqlCdcSources.Builder initialSourceBuilder() {
+        return MySqlCdcSources.mysql("cdcMysql")
+                .setDatabaseAddress(mysql.getContainerIpAddress())
+                .setDatabasePort(mysql.getMappedPort(MYSQL_PORT))
+                .setDatabaseUser("debezium")
+                .setDatabasePassword("dbz")
+                .setClusterName("dbserver1");
+    }
+
+    protected Pipeline preparePipeline(StreamSource<ChangeRecord> source) {
+        Pipeline pipeline = Pipeline.create();
+        pipeline.readFrom(source)
+                .withNativeTimestamps(0)
+                .filter(t -> t.database().startsWith(DATABASE))
+                .setLocalParallelism(1)
```

Comment:
No need to set local parallelism, the source by default has a single instance.

Reply:
actually, by default local parallelism for `filter` is `4`:
```
digraph DAG {
	"cdcMysql" [localParallelism=1];
	"filter" [localParallelism=4];
	"filter_timestamps" [localParallelism=1];
        ...
```